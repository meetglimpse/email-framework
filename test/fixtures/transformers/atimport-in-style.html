<!doctype html>
<html>
  <head>
    <style postcss>
      @import "test/stubs/post.css";
    </style>
  </head>
  <body>
    <div>test</div>
  </body>
</html>
