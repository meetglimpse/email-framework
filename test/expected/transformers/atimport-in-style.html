<!doctype html>
<html>
  <head>
    <style>div {
  margin-top: 1px;
  margin-right: 2px;
  margin-bottom: 3px;
  margin-left: 4px;
}
    </style>
  </head>
  <body>
    <div>test</div>
  </body>
</html>
